js/version.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+interface Version {
+  deno: string;
+  v8: string;
+  typescript: string;
+}
+
+export const version: Version = {
+  deno: "",
+  v8: "",
+  typescript: "TS_VERSION" // This string will be replaced by rollup
+};
+
+/**
+ * Sets the deno and v8 versions and freezes the version object.
```

Comment:
I would set this to `@internal` so it doesn't get leaked to the runtime type library.